cmd/soroban-rpc/internal/preflight/preflight_test.go

```diff
@@ -330,6 +330,7 @@ func TestGetPreflight(t *testing.T) {
 	params := getPreflightParameters(t, nil)
 	_, err := GetPreflight(context.Background(), params)
 	require.NoError(t, err)
+	params.LedgerEntryReadTx.Done()
```

Comment:
could you change this to
```golang
require.NoError(b, params.LedgerEntryReadTx.Done())
```
in a similar way you've done in other places ?

Reply:
Sure